common/source/docs/common-blheli32-passthru.rst

```diff
@@ -1,62 +1,149 @@
 .. _common-blheli32-passthru:
 
-BLHeli32/ BLHeli-S Pass-Through Support
-=======================================
+BLHeli32 and BLHeli_S ESCs
+==========================
 
-In addition to offering enhanced capabilities such as reversing and often, telemetry, BLHeli capable ESCs have the ability to have their parameters and features programmed through the autopilot while it is connected to a PC (if the autopilot firmware provides this capability also). This BLHeli pass-through protocol allows you to configure and upgrade your ESCs without having to disconnect them from your vehicle. You can plug a USB cable into your autopilot and run the BLHeliSuite or BLHeliSuite32 software for Windows to configure your ESCs. 
+The BLHeli protocol is built on the :ref:`DShot <common-dshot-escs>` protocol and offers these additional features
 
-Most BLHeli ESCs can autodetect if PWM or DShot is selected by the motor protocol parameter set by the user. See the configuration section of :ref:`common-brushless-escs` for more information. In addition, some BLHeli32 capable ESCs offer bi-directional capabilities for telemetry reporting over the control link, see the section below.
+- Pass-Through support the BLHeli application to be used to configure the ESCs while remaining connected to the autopilot
+- :ref:`Reversible DShot <blheli32-reversible-dshot>` meaning the motor can be spun in either direction
```

Comment:
this is commonly called "3D mode"...should be mentioned

Reply:
yes, totally agree. added "aka 3D mode".